### PR TITLE
update changelog for v0.8.0-rc.1

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+#### [v0.8.0-rc.1](https://github.com/heptio/ark/releases/tag/v0.8.0-rc.1) - 2018-04-11
+
+Highlights:
+  * Backup deletion has been completely revamped to make it simpler and less error-prone. As a user, you still use the `ark backup delete` command to request deletion of a backup and its associated cloud
+  resources; behind the scenes, we've switched to using a new `DeleteBackupRequest` Custom Resource and associated controller for processing deletion requests.
+  * We've reduced the number of required fields in the Ark config. For Azure, `location` is no longer required, and for GCP, `project` is not needed.
+  * Ark now copies tags from source volumes to snapshots during backup, and from source snapshots to new volumes during restore.
```
Comment (ncdc) [inline CHANGELOG.md]: Maybe remove "source"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+  resources; behind the scenes, we've switched to using a new `DeleteBackupRequest` Custom Resource and associated controller for processing deletion requests.
+  * We've reduced the number of required fields in the Ark config. For Azure, `location` is no longer required, and for GCP, `project` is not needed.
+  * Ark now copies tags from source volumes to snapshots during backup, and from source snapshots to new volumes during restore. 
+
+Breaking Changes:
+  * Ark has moved back to a single namespace (`heptio-ark` by default) as part of #383.
+
+All New Features:
+  * Add global `--kubecontext` flag to Ark CLI (#296, @blakebarnett)
+  * Azure: support cross-resource group restores of volumes (#356 #378 @skriss)
```
Comment (ncdc) [inline CHANGELOG.md]: Add a comma before your handle

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+  * AWS/Azure/GCP: copy tags from volumes to snapshots, and from snapshots to volumes (#341, @skriss)
+  * Replace finalizer for backup deletion with `DeleteBackupRequest` custom resource & controller (#383, @ncdc)
+  * Don't log warnings during restore if an identical object already exists in the cluster (#405 @nrb)
```
Comment (ncdc) [inline CHANGELOG.md]: Add a comma before nrb

**Comment by skriss** [comment]
I was thinking we wouldn't leave the RC section around -- just update the whole thing to v0.8.0 once we're through the RCs

**Comment by ncdc** [comment]
That could work. What do you recommend, @Bradamant3?

**Comment by skriss** [comment]
addressed other comments

**Comment by Bradamant3** [comment]
Agree, let's update the whole thing after we're done with the RCs

**Pull request merged**